Formula/b/bazel.rb

```diff
@@ -31,9 +31,9 @@ def install
     ENV["EMBED_LABEL"] = "#{version}-homebrew"
     # Force Bazel ./compile.sh to put its temporary files in the buildpath
     ENV["BAZEL_WRKDIR"] = buildpath/"work"
-    # Force Bazel to use openjdk@11
+    # Force Bazel to use openjdk@21
     ENV["EXTRA_BAZEL_ARGS"] = "--host_javabase=@local_jdk//:jdk"
-    ENV["JAVA_HOME"] = Language::Java.java_home("11")
+    ENV["JAVA_HOME"] = Language::Java.java_home("21")
```

Comment:
There is an issue in the bazel project which mentioned the build args changed https://github.com/bazelbuild/bazel/issues/22789

It appears that `--tool_java_runtime_version=local_jdk` is the correct arg now https://github.com/easybuilders/easybuild-easyblocks/blob/064e3e24b4638ed2e20d8ecd3370c2ae459aee68/easybuild/easyblocks/b/bazel.py#L169